core/src/main/java/tech/pegasys/eth2signer/core/multikey/metadata/FileRawSigningMetadata.java

```diff
@@ -12,26 +12,28 @@
  */
 package tech.pegasys.eth2signer.core.multikey.metadata;
 
-import tech.pegasys.eth2signer.core.multikey.MultiSignerFactory;
 import tech.pegasys.eth2signer.core.signing.ArtifactSigner;
 
-import org.apache.tuweni.bytes.Bytes;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class UnencryptedKeyMetadataFile extends SigningMetadataFile {
+public class FileRawSigningMetadata implements SigningMetadata {
 
-  final Bytes privateKeyBytes;
+  private final String privateKey;
 
-  public UnencryptedKeyMetadataFile(final String filename, final Bytes privateKeyBytes) {
-    super(filename);
-    this.privateKeyBytes = privateKeyBytes;
+  @JsonCreator
+  public FileRawSigningMetadata(
+      @JsonProperty(value = "privateKey", required = true) final String privateKey) {
+    this.privateKey = privateKey;
   }
 
-  public Bytes getPrivateKeyBytes() {
-    return privateKeyBytes;
+  @JsonProperty(value = "privateKey")
+  public String getPrivateKey() {
+    return privateKey;
   }
 
   @Override
-  public ArtifactSigner createSigner(final MultiSignerFactory factory) {
+  public ArtifactSigner createSigner(final ArtifactSignerFactory factory) {
```

Comment:
? still doing double dispatch?

Reply:
Yeah it's simpler to test with the double dispatch in place, but I can get rid of it.

Reply:
got rid it